### PR TITLE
A0-1664 Stop exposing sync connected events from substrate network

### PR DESCRIPTION
**Pull request by lesniak43**
https://cardinal-cryptography.atlassian.net/browse/A0-1664

### CODE REVIEW

File: finality-aleph/src/network/mock.rs [hunk 1]
```diff
@@ -182,17 +175,11 @@ pub struct MockNetwork {
 }
 
 #[derive(Debug, Copy, Clone)]
-pub enum MockSenderError {
-    SomeError,
-}
+pub struct MockSenderError();
```
Comment (timorl) [inline finality-aleph/src/network/mock.rs:178]: I believe these parentheses are unnecessary.

**Review by timorl — APPROVED** [review]
Looks good as long as there is a plan to reacquire these tests.

File: finality-aleph/src/network/mock.rs (hunk 1 above)
Comment (lesniak43) [reply finality-aleph/src/network/mock.rs]: indeed

**Review by maciejnems — APPROVED** [review]
Rebased on top of main, so tests are no longer removed.

**Pull request merged**